producers/cube_config_mfx.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+
+# Custom bins
+def binBoundaries(run):
+    if isinstance(run,str):
+        run=int(run)
+    if run>0:
+        # return np.arange(-5.,50.,0.2)
+        return np.arange(-5.,10.,1)
+    return None
+
+
+# ##### FILTERS ##########################################################################
+# filters to apply to the data
+# format: list of [det (field), low, high, name]
+# 'filter1' is the standard name and will not be added to the h5 file name.
+filters = [
+    ['lightStatus/xray',0.5,1.5,'filter1'],
+    ['ipm_dg1/sum',1e3,4.5e4,'filter1'],
+    # ['ipm3/sum',200,3e3,'filter1'],
+    # ['tt/FLTPOSFWHM',60,230,'filter1'],
+    # ['tt/AMPL',0.005,0.19,'filter1']
+    # ['evr/code_41',0.5,1.5,'custom']
+]
+
+# Laser on/off
+laser = False
+
+# ##### DETECTORS ########################################################################
+# List detectors to be cubed. Area detector have additional options such as threshold
+# Full images will always be saved. 
+# More area detector can be defined following the same syntax, and adding them to varList
+detDict = {'source':'Rayonix', 
+           'full':1, 
+           'image':1, 
+           'thresADU':-1e5, 
+           'common_mode':0}
+
+pix_size = 176e-6
```

Comment:
Is that where it needs to be and not in the func arguments?

Reply:
This was just for me to test with a rough estimate of the central position in the Rayonix. The center argument should be in meter, so this is just my way of cheating into it.